site/app/views/admin/UsersView.php

```diff
@@ -61,7 +61,6 @@ public function listGraders($graders, $reg_sections, $rot_sections, $use_databas
             "reg_sections" => $reg_sections,
             "rot_sections" => $rot_sections,
             "use_database" => $use_database,
-            "return_url_update_grader" => $this->core->buildUrl(array('component' => 'admin', 'page' => 'users','action' => 'update_grader')),
```

Comment:
Shifting this to `userForm` should help!

Reply:
Might need some modification though

Reply:
I switched it back to using buildUrl in the twig file, its more complicated then I thought so trying to fix it as I passed through the files was a mistake.  This should be changed in the future within #2273, but for now changes are just to make it work as before.